recipes/visp/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set name = "visp" %}
+{% set version = "3.5.0" %}
+{% set major_minor_version = version.split('.')[0] + version.split('.')[1] + version.split('.')[2] %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lagadic/visp/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 494a648b2570da2a200ba326ed61a14e785eb9ee08ef12d3ad178b2f384d3d30
+  patches:
+    - patches/0001-Fix-invalid-usage-of-OGRE_DIR-in-CMakeLists.txt.patch
+    - patches/0002-Disable-AVX-and-AVX2-for-Simd-embedded-lib.patch  # [win]
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - clang  # [win]
+    - make  # [not win]
+    - ninja  # [win]
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxext') }}  # [linux]
```

Nit: Yes, I am curing my OCD.
```suggestion
    - make                           # [not win]
    - clang                          # [win]
    - ninja                          # [win]
    - {{ cdt('mesa-libgl-devel') }}  # [linux]
    - {{ cdt('mesa-dri-drivers') }}  # [linux]
    - {{ cdt('libselinux') }}        # [linux]
    - {{ cdt('libxdamage') }}        # [linux]
    - {{ cdt('libxxf86vm') }}        # [linux]
    - {{ cdt('libxext') }}           # [linux]
```

---

Agreed, this is much more readable. I thought linting rules were fixed to two spaces before he platform specification, but this seems OK for the linter. Will do.